### ISSUE TITLE
[FR] Update similarity view stage in-place when multiple consecutive searches are performed

### ISSUE THREAD

**Issue by brimoor** [voxel51/fiftyone#1223]
When working with visual similarity from the App, I find myself doing the click image + sort by similarity workflow many times. Currently, each time results in a new view stage, which starts to get out-of-hand in the view bar.

Instead, it would be nice if consecutive searches with the same `k` value did not result in a new stage, since the first sort is completely unnecessary in that case.

```py
import fiftyone as fo
import fiftyone.brain as fob
import fiftyone.zoo as foz

dataset = foz.load_zoo_dataset("quickstart")
fob.compute_similarity(dataset, brain_key="image_sim")

# Do multiple image similarity sorts
session = fo.launch_app(dataset)
```

<img width="1273" alt="Screen Shot 2021-08-24 at 11 57 39 PM" src="https://user-images.githubusercontent.com/25985824/130724915-4430e0c3-0597-4bbc-b2ff-fb9d4246ca8d.png">

**Comment by cavesdev** [comment]
Hello! Is someone already working on this?

**Comment by benjaminpkane** [comment]
@cavesdev no, this is open. Feel free to work on this.

**Comment by brimoor** [comment]
This has been addressed as of `fiftyone==0.20`!
